src/twisted/web/test/test_domhelpers.py

```diff
@@ -122,15 +122,16 @@ def test_clearNode(self):
 
     def test_get(self):
         doc1 = self.dom.parseString('<a><b id="bar"/><c class="foo"/></a>')
+        doc = self.dom.Document()
         node=domhelpers.get(doc1, "foo")
         actual=node.toxml()
```

Comment:
since you're fixing spaces around assignments, do it here as well? (and perhaps other places in the test functions)